<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>CScreenShareToolWgd</class>
 <widget class="QWidget" name="CScreenShareToolWgd">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>360</width>
    <height>111</height>
   </rect>
  </property>
  <property name="minimumSize">
   <size>
    <width>360</width>
    <height>111</height>
   </size>
  </property>
  <property name="maximumSize">
   <size>
    <width>360</width>
    <height>111</height>
   </size>
  </property>
  <property name="windowTitle">
   <string>CScreenShareToolWgd</string>
  </property>
  <property name="styleSheet">
   <string notr="true"/>
  </property>
  <layout class="QVBoxLayout" name="verticalLayout">
   <property name="spacing">
    <number>0</number>
   </property>
   <property name="leftMargin">
    <number>0</number>
   </property>
   <property name="topMargin">
    <number>0</number>
   </property>
   <property name="rightMargin">
    <number>0</number>
   </property>
   <property name="bottomMargin">
    <number>0</number>
   </property>
   <item>
    <widget class="QFrame" name="screenShareTopFrame">
     <property name="minimumSize">
      <size>
       <width>0</width>
       <height>70</height>
      </size>
     </property>
     <property name="maximumSize">
      <size>
       <width>16777215</width>
       <height>70</height>
      </size>
     </property>
     <property name="frameShape">
      <enum>QFrame::StyledPanel</enum>
     </property>
     <property name="frameShadow">
      <enum>QFrame::Raised</enum>
     </property>
     <layout class="QGridLayout" name="gridLayout">
      <property name="leftMargin">
       <number>20</number>
      </property>
      <property name="topMargin">
       <number>5</number>
      </property>
      <property name="rightMargin">
       <number>20</number>
      </property>
      <property name="bottomMargin">
       <number>0</number>
      </property>
      <property name="spacing">
       <number>0</number>
      </property>
      <item row="0" column="0">
       <widget class="ToolButton" name="pushButton_camera">
        <property name="minimumSize">
         <size>
          <width>44</width>
          <height>37</height>
         </size>
        </property>
        <property name="maximumSize">
         <size>
          <width>44</width>
          <height>37</height>
         </size>
        </property>
        <property name="styleSheet">
         <string notr="true"/>
        </property>
        <property name="text">
         <string>摄像头</string>
        </property>
       </widget>
      </item>
      <item row="0" column="1">
       <widget class="Line" name="line_5">
        <property name="minimumSize">
         <size>
          <width>1</width>
          <height>0</height>
         </size>
        </property>
        <property name="maximumSize">
         <size>
          <width>1</width>
          <height>16777215</height>
         </size>
        </property>
        <property name="styleSheet">
         <string notr="true">border-image: url(:/interactivity/img/interactivity/line.png);</string>
        </property>
        <property name="orientation">
         <enum>Qt::Vertical</enum>
        </property>
       </widget>
      </item>
      <item row="0" column="2">
       <widget class="ToolButton" name="pushButton_mic">
        <property name="minimumSize">
         <size>
          <width>33</width>
          <height>39</height>
         </size>
        </property>
        <property name="maximumSize">
         <size>
          <width>33</width>
          <height>39</height>
         </size>
        </property>
        <property name="text">
         <string>麦克风</string>
        </property>
       </widget>
      </item>
      <item row="0" column="3">
       <widget class="Line" name="line_6">
        <property name="minimumSize">
         <size>
          <width>1</width>
          <height>0</height>
         </size>
        </property>
        <property name="maximumSize">
         <size>
          <width>1</width>
          <height>16777215</height>
         </size>
        </property>
        <property name="styleSheet">
         <string notr="true">border-image: url(:/interactivity/img/interactivity/line.png);</string>
        </property>
        <property name="orientation">
         <enum>Qt::Vertical</enum>
        </property>
       </widget>
      </item>
      <item row="0" column="4">
       <widget class="ToolButton" name="tBtnQuikTool">
        <property name="minimumSize">
         <size>
          <width>33</width>
          <height>39</height>
         </size>
        </property>
        <property name="maximumSize">
         <size>
          <width>33</width>
          <height>39</height>
         </size>
        </property>
        <property name="text">
         <string>工具</string>
        </property>
       </widget>
      </item>
      <item row="0" column="5">
       <widget class="Line" name="line_7">
        <property name="minimumSize">
         <size>
          <width>1</width>
          <height>0</height>
         </size>
        </property>
        <property name="maximumSize">
         <size>
          <width>1</width>
          <height>16777215</height>
         </size>
        </property>
        <property name="styleSheet">
         <string notr="true">border-image: url(:/interactivity/img/interactivity/line.png);</string>
        </property>
        <property name="orientation">
         <enum>Qt::Vertical</enum>
        </property>
       </widget>
      </item>
      <item row="0" column="6">
       <widget class="ToolButton" name="pushButton_startLive">
        <property name="minimumSize">
         <size>
          <width>44</width>
          <height>37</height>
         </size>
        </property>
        <property name="maximumSize">
         <size>
          <width>44</width>
          <height>37</height>
         </size>
        </property>
        <property name="text">
         <string>开始直播</string>
        </property>
       </widget>
      </item>
      <item row="0" column="7">
       <widget class="Line" name="line_startLive">
        <property name="minimumSize">
         <size>
          <width>1</width>
          <height>0</height>
         </size>
        </property>
        <property name="maximumSize">
         <size>
          <width>1</width>
          <height>16777215</height>
         </size>
        </property>
        <property name="styleSheet">
         <string notr="true">border-image: url(:/interactivity/img/interactivity/line.png);</string>
        </property>
        <property name="orientation">
         <enum>Qt::Vertical</enum>
        </property>
       </widget>
      </item>
      <item row="0" column="8">
       <widget class="ToolButton" name="pushButton_player">
        <property name="minimumSize">
         <size>
          <width>44</width>
          <height>38</height>
         </size>
        </property>
        <property name="maximumSize">
         <size>
          <width>44</width>
          <height>37</height>
         </size>
        </property>
        <property name="text">
         <string>扬声器</string>
        </property>
       </widget>
      </item>
     </layout>
    </widget>
   </item>
   <item>
    <widget class="QFrame" name="screenShareButtomFrame">
     <property name="frameShape">
      <enum>QFrame::StyledPanel</enum>
     </property>
     <property name="frameShadow">
      <enum>QFrame::Raised</enum>
     </property>
     <layout class="QHBoxLayout" name="horizontalLayout">
      <property name="spacing">
       <number>0</number>
      </property>
      <property name="leftMargin">
       <number>0</number>
      </property>
      <property name="topMargin">
       <number>0</number>
      </property>
      <property name="rightMargin">
       <number>0</number>
      </property>
      <property name="bottomMargin">
       <number>20</number>
      </property>
      <item>
       <widget class="ToolButton" name="pushButton_stopShare">
        <property name="minimumSize">
         <size>
          <width>43</width>
          <height>13</height>
         </size>
        </property>
        <property name="maximumSize">
         <size>
          <width>43</width>
          <height>13</height>
         </size>
        </property>
        <property name="styleSheet">
         <string notr="true"/>
        </property>
        <property name="text">
         <string>停止共享功能</string>
        </property>
       </widget>
      </item>
     </layout>
    </widget>
   </item>
  </layout>
 </widget>
 <layoutdefault spacing="6" margin="11"/>
 <customwidgets>
  <customwidget>
   <class>ToolButton</class>
   <extends>QPushButton</extends>
   <header location="global">ToolButton.h</header>
  </customwidget>
 </customwidgets>
 <resources>
  <include location="../../VhallLive/project/VhallLive.qrc"/>
 </resources>
 <connections/>
</ui>
